packages/ETLoad.package/ETLDirectoryTestCase.class/instance/doesNotUnderstand..st

```diff
@@ -0,0 +1,9 @@
+private
+doesNotUnderstand: aMessage
+	"intercept test messages and forward them to #testFile with
+	the file name extracted from the message selector."
+
+	(aMessage selector startsWith: 'test') ifTrue: [
+		^ self testFile: (aMessage selector allButFirst: 4). ].
+	
+	^ super doesNotUnderstand: aMessage.
```

Comment:
Well, the class name already contains the kind of test, in this case `ETLSampleImageSegmentTest`.